controllers/front/ProductController.php

```diff
@@ -106,11 +106,17 @@ public function init()
 
         $id_product = (int) Tools::getValue('id_product');
 
+        // Get product by EAN-13
+        $ean13_product = Tools::getValue('ean13_product');
+        if (!is_null($ean13_product)) {
```

Comment:
maybe we could check that don't already have an `$id_product` to make sure we don't perform a useless SQL query?